bentoml/_internal/log.py

```diff
@@ -71,19 +71,21 @@ def format(self, record):
     },
     "handlers": {
         "internal": {
-            "level": "INFO",
+            "level": "DEBUG" if get_debug_mode() else "INFO",
```

Comment:
I don't think this is needed, it will be overridden in the cli

Reply:
sg